cmd/sojuctl/main.go

```diff
@@ -107,3 +105,30 @@ func main() {
 		}
 	}
 }
+
+func readPassword() ([]byte, error) {
+	var password []byte
+	var err error
+	fd := int(os.Stdin.Fd())
+
+	if terminal.IsTerminal(fd) {
+		fmt.Printf("Password: ")
+		password, err = terminal.ReadPassword(int(os.Stdin.Fd()))
+		if err != nil {
+			return nil, err
+		}
+		fmt.Printf("\n")
+	} else {
+		stdin, err := ioutil.ReadAll(os.Stdin)
```

Comment:
print a warning to stderr when reading password from stdin (so that users who run in a non-functional terminal can see that they must input a password)